{
  "dependencies": {
    "python_class": "serializable_elist", 
    "python_module": "g_sorcery.g_collections", 
    "value": {
      "data": [], 
      "separator": "\n\t"
    }
  }, 
  "description": "Django view that raises an exception so you can test your logging setup", 
  "eclasses": [
    "g-sorcery", 
    "gs-pypi"
  ], 
  "homepage": "http://github.com/brightinteractive/django-raise-exception-view/", 
  "info": {
    "Author:": "Bright Interactive", 
    "Home Page:": "http://github.com/brightinteractive/django-raise-exception-view/", 
    "License:": "BSD", 
    "Package Index Owner:": "bright"
  }, 
  "license": "", 
  "longdescription": "Django view that raises an exception so you can test your logging setup", 
  "maintainer": [
    {
      "email": "piatlicki@gmail.com", 
      "name": "Jauhien Piatlicki"
    }
  ], 
  "md5": "88e2eb0202a60f790c00cf1986feaa8f", 
  "source_uri": "https://pypi.python.org/packages/source/d/django-raise-exception-view/django-raise-exception-view-1.0.1.tar.gz"
}